src/Zettle/README.md

```diff
@@ -0,0 +1,66 @@
+# Zettle
+
+Zettle by Paypal is a point of sale payment provider. [API documentation](https://developer.zettle.com/docs/api)
+
+```bash
+composer require socialiteproviders/zettle
+```
+
+## Installation & Basic Usage
+
+Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
+
+### Add configuration to `config/services.php`
+
+```php
+'zettle' => [
+  'client_id' => env('ZETTLE_CLIENT_ID'),
+  'client_secret' => env('ZETTLE_CLIENT_SECRET'),
+  'redirect' => env('ZETTLE_REDIRECT_URI')
+],
+```
+
+### Add provider event listener
+
+#### Laravel 11+
+
+In Laravel 11, the default `EventServiceProvider` provider was removed. Instead, add the listener using the `listen` method on the `Event` facade, in your `AppServiceProvider` `boot` method.
+
+* Note: You do not need to add anything for the built-in socialite providers unless you override them with your own providers.
+
+```php
+Event::listen(function (\SocialiteProviders\Manager\SocialiteWasCalled $event) {
+    $event->extendSocialite('zettle', \SocialiteProviders\Zettle\Provider::class);
+});
+```
+<details>
+<summary>
+Laravel 10 or below
+</summary>
+Configure the package's listener to listen for `SocialiteWasCalled` events.
+
+Add the event to your `listen[]` array in `app/Providers/EventServiceProvider`. See the [Base Installation Guide](https://socialiteproviders.com/usage/) for detailed instructions.
+
+```php
+protected $listen = [
+    \SocialiteProviders\Manager\SocialiteWasCalled::class => [
+        // ... other providers
+        \SocialiteProviders\Zettle\ZettleExtendSocialite::class.'@handle',
+    ],
+];
+```
+</details>
+
+### Usage
+
+You should now be able to use the provider like you would regularly use Socialite (assuming you have the facade installed):
```

Comment:
Can you add a comment to mention that the provider only returns a uuid id?

Reply:
I added a section about it. [According to the docs](https://developer.zettle.com/docs/api/oauth/api-reference#responses-7) you only get a user ID and organization ID. There is no scope to get more details AFAIK.